### PR TITLE
Put trailhead feture list on the side

### LINKED ISSUE #1241: (Trailhead) Move Product List to the right of the registration form
Context:
Yesterday it was determined that the newsletters needed to be moved to the first screen of registration flow due to API limitations (among other things).

Yesterday night, it was proposed to just make a very long layout but there are usability problems there so UX proposed the following layout this morning.

Screen Shot 2019-05-23 at 10 22 38 AM

Acknowledgment, the password strength bubble will hide the product list. We believe that this is OK since people will not be reading about products while picking a password.

CC @shane-tomlinson @ryanfeeley

### PR DESCRIPTION
**Pull request by dannycoates**
Still need to do `index` and `choose_what_to_sync` templates. How do I browse them locally?

Closes #1241

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
>Still need to do index and choose_what_to_sync templates. How do I browse them locally?

Make sure you have a profile set up for local development. I use [https://github.com/vladikoff/fxa-dev-launcher/] and type `FXA_ENV=local npm start`

* index: http://127.0.0.1:3030/?action=email&context=fx_desktop_v3&service=sync&style=trailhead
* CWTS: Enter an email on index, add a password on the next page, and hit submit
.

**Comment by dannycoates** [comment]
Screen Shot 2019-05-24 at 4 37 16 PM
Screen Shot 2019-05-24 at 4 38 19 PM
Screen Shot 2019-05-24 at 4 44 20 PM

**Comment by dannycoates** [comment]
@shane-tomlinson r?

File: packages/fxa-content-server/app/styles/modules/_branding.scss [hunk 1]
```diff
@@ -72,6 +72,7 @@
   h2 {
     font-size: 18px;
     font-weight: 600;
+    line-height: 1.5rem;
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/app/styles/modules/_branding.scss]: TIL. `rem` looks amazing.

**Comment by dannycoates** [comment]
I'll do a follow up to add a breakpoint for the tablet width

**Pull request merged**